module/modmanager/manager.go

```diff
@@ -332,23 +345,25 @@ func (mgr *Manager) Reconfigure(ctx context.Context, conf config.Module) ([]reso
 	mod.cfg = conf
 	mod.resources = map[resource.Name]*addedResource{}
 
+	mgr.logger.CInfow(ctx, "Existing module process stopped. Starting new module process", "module", conf.Name)
+
 	if err := mgr.startModule(ctx, mod); err != nil {
 		// If re-addition fails, assume all handled resources are orphaned.
 		return handledResourceNames, err
 	}
 
-	mgr.logger.Debugw("successfully reconfigured and reconnected to module", "module", mod.cfg.Name, "module address", mod.addr)
+	mgr.logger.CDebugw(ctx, "New module process is running and responding to gRPC requests", "module", mod.cfg.Name, "module address", mod.addr)
 
 	// add old module process' resources to new module; warn if new module cannot
 	// handle old resource and consider that resource orphaned.
 	var orphanedResourceNames []resource.Name
 	for name, res := range handledResources {
 		if _, err := mgr.addResource(ctx, res.conf, res.deps); err != nil {
-			mgr.logger.Warnf("error while re-adding resource %s to module %s: %v",
-				name, conf.Name, err)
+			mgr.logger.CWarnw(ctx, "Error while re-adding resource to module",
```

Comment:
I'm generally preferring the `C` and `w` versions of log methods where possible and capitalizing log messages. @dgottlieb and I spoke offline a bit, and it seems like we prefer structured logging rather than formatted `f` logging where possible, as log messages are easier to grep for when trying to debug complex issues. Furthermore, I think we should stylistically make some sort of decision on upper-casing or lower-casing logs. I think Dan and I would both vote for upper-case, as some log messages are multiple sentences (`"This event happened. Now doing this"`), and multiple sentences with lower-cased starts are harder to parse. If we agree on using `C` and `w` versions of logging methods where possible with upper-cased messages, I'll file a ticket to standardize more of the NetCode stack to have that pattern. Starting with all the logs in the module manager and some logs in local robot/resource manager seems good.

Reply:
cc @viamrobotics/netcode .

Reply:
yep, the format/pattern sounds good to me

Reply:
https://viam.atlassian.net/browse/RSDK-7050.